rest/utilities_testing.go

```diff
@@ -381,6 +434,26 @@ func (rt *RestTester) GetDatabase() *db.DatabaseContext {
 	return nil
 }
 
+// GetSingleTestDatabaseCollection will return a DatabaseCollection if there is only one. Depending on test environment configuration, it may or may not be the default collection.
+func (rt *RestTester) GetSingleTestDatabaseCollection() *db.DatabaseCollection {
+	database := rt.GetDatabase()
+	require.Equal(rt.TB, 1, len(database.CollectionByID))
+	for _, collection := range database.CollectionByID {
+		return collection
+	}
+	return nil
+}
+
+// GetSingleTestDataStore will return a datastore if there is only one collection configured on the RestTester database.
+func (rt *RestTester) GetSingleTestDataStore() base.DataStore {
```

Comment:
Maybe a nitpick, but what do you think about calling this GetSingleDataStore()?  

The current name had me thinking there was a struct TestDataStore (similar to TestBucket), and since this is a method on RestTester I think the 'Test' is redundant here.

Reply:
I feel OK about this, but I used this to clarify that these were OK to use because it is explicit test usage where I've made the change. I think we have to evaluate if we are using it in a non test case with a grep, and I think this is fine for `RestTester` because the bugs would be test only.